src/test/java/com/snack/news/service/AdminServiceTest.java

```diff
@@ -1,22 +1,25 @@
 package com.snack.news.service;
 
 import com.snack.news.domain.news.News;
-import com.snack.news.exception.CategoryNotFoundException;
-import com.snack.news.exception.NewsNotFoundException;
-import com.snack.news.exception.TagNotFoundException;
-import com.snack.news.exception.TopicNotFoundException;
+import com.snack.news.domain.picks.Pick;
+import com.snack.news.dto.PickDto;
+import com.snack.news.exception.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.snack.news.exception'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)